### DISCUSSION TITLE
Dropdown element

### DISCUSSION THREAD

**Discussion by Phonix88** [elimintz/justpy#326] [Q&A] [answered]
hello everyone, I'm setting out to discover this incredible library.  although its development is already very advanced, I have not found the drop down menus.  They are not implemented yet I have not found them?  By drop-down menu, I mean this:

https://getbootstrap.com/docs/4.0/components/dropdowns/

**Comment by elimintz** [comment, marked as answer]
The quickest way to get dropdown functionality is to use the Quasar elements with JustPy:
https://justpy.io/quasar_tutorial/introduction/

The specific element you want is: https://v1.quasar.dev/vue-components/button-dropdown#qbtndropdown-api

You can see it being used in this example:  https://github.com/elimintz/justpy/blob/master/examples/dogs.py#L69

**Reply by Phonix88** [reply]
Great!
I take this opportunity to congratulate you on the development of this library. For some time I have been looking for one with such a pythonic approach.
